api-report/telemetry-utils.api.md

```diff
@@ -23,9 +23,13 @@ import { TypedEventEmitter } from '@fluidframework/common-utils';
 
 // @public
 export class ChildLogger extends TelemetryLogger {
+    static create(baseLogger?: ITelemetryBaseLogger, namespace?: string, properties?: ITelemetryLoggerPropertyBags, globalProperties?: ITelemetryLoggerPropertyBags, layerVersion?: string): TelemetryLogger;
```

Comment:
I do not think base layer needs to know anything about concept of layers or versions. I'd prefer these concepts not to be present in base layer (and if needed, to be in add-on layer).